utility/src/test/TestManifest.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="org.oppia.util">
+  <uses-sdk android:minSdkVersion="16"
```

Comment:
Shouldn’t min sdk be 19? We don’t support JB.

Reply:
Yes! Will change now.